### ISSUE TITLE
Vue CLI 4 add Quasar Plugin failed

### ISSUE THREAD

**Issue by hugetiny** [quasarframework/quasar#5716]
**Describe the bug**
Vue CLI 4 add Quasar Plugin failed.
```
error: Unnecessary escape character: \/ (no-useless-escape) at vue.config.js:9:39:
   7 |   },
   8 |   transpileDependencies: [
>  9 |     /[\\\/]node_modules[\\\/]quasar[\\\/]/
     |                                       ^
  10 |   ]
  11 | }
  12 |

```

**Codepen/jsFiddle/Codesandbox (required)**
Vue CLI 4 Quasar Plugin mode

**To Reproduce**
1.  vue create my-app
2. cd my-app
3. vue add quasar

**Screenshots**
![image](https://user-images.githubusercontent.com/19659202/69775123-c38fb180-11d2-11ea-8c89-18950d5c0364.png)

**Platform (please complete the following information):**
OS:Windows10
Node:10.16.3
NPM:6.9.0
Yarn:1.19
Browsers:Chrome
iOS:-
Android:-
Electron:-

**Comment by hugetiny** [comment]
vue --version
@vue/cli 4.0.4

**Comment by hugetiny** [comment]
@vue/cli 3.12.1 is ok to run

**Comment by yusufkandemir** [comment]
Duplicate of https://github.com/quasarframework/vue-cli-plugin-quasar/issues/20

**Comment by yusufkandemir** [comment]
First of all this issue doesn't relate to this repository, see the comment above. The fix is implemented in [Quasar Vue CLI Plugin v2.0.0](https://github.com/quasarframework/vue-cli-plugin-quasar/releases/tag/v2.0.0), with [this commit](https://github.com/quasarframework/vue-cli-plugin-quasar/commit/c4c56fb741c1b2e8c2e63c1485387be9d47adba2).
@rstoenescu I think this issue can be closed.
